common/resource.go

```diff
@@ -300,6 +300,51 @@ func WorkspaceData[T any](read func(context.Context, *T, *databricks.WorkspaceCl
 	}
 }
 
+func AccountData[T any](read func(context.Context, *T, *databricks.AccountClient) error) *schema.Resource {
```

Comment:
We don't have any test for WorkspaceData. I assume it's because they are supposed to be tested when used in other resources and resources tests. Similar would be for AccountData.

It will be tested in: https://github.com/databricks/terraform-provider-databricks/pull/2017